src/utils/getListLabel.ts

```diff
@@ -0,0 +1,7 @@
+import { splitOnUppercaseChars } from "./splitOnUppercaseChars";
```

Comment:
I noticed that the utility files are called the same way as the functions within them. Usually one utility file contains multiple functions inside. So the names of the files should be more generic. For instance, the current file deals with string transformation, so we can call it strings.ts.

Reply:
Resolved.